src/Admin/CommentAdmin.php

```diff
@@ -18,7 +18,7 @@
 use Sonata\AdminBundle\Datagrid\ListMapper;
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Form\Type\ModelListType;
-use Sonata\CoreBundle\Model\ManagerInterface;
+use Sonata\Doctrine\Model\ManagerInterface;
```

Comment:
This is used in a signature, and signatures will not trigger autoload. Please trigger manually as described in the last paragraph of my blog post I recently added.

Reply:
Haven't done this on the other `Removed CoreBundle deprecations` PRs either...

Reply:
I also didn't deprecate anything in this PR. I removed the deprecations caused be the CoreBundle

Reply:
ping @greg0ire

Reply:
Yeah those are not used in type hints in most cases, unlike here.

Reply:
So, what to do?

Reply:
This looks wrong. This trigger should be added to the `doctrine-extensions` class. 

In fact there is a trigger for this: https://github.com/sonata-project/sonata-doctrine-extensions/blob/1.x/src/Model/ManagerInterface.php#L101

Reply:
There is indeed, but that file will not be loaded because of a type hint. Type hints do not trigger autoload.

Reply:
Wait no it seems to work :thinking:

Reply:
I don't get why, the parent interface gets autoloaded by something… guess things are fine then